src/redux/configureStore.js

```diff
@@ -0,0 +1,23 @@
+import { applyMiddleware, configureStore } from '@reduxjs/toolkit';
+import thunk from 'redux-thunk';
+import userReducer from './user/userSlice';
+import details from './details/GetDetails';
+// import properties from './properties/GetProperties';
```

Comment:
- [ ] Kindly, remove this old piece of commented code in order to improve the repo's cleanliness. 👍